### ISSUE TITLE
export.py

### ISSUE THREAD

**Issue by ccf854** [yuyun2000/SpeechDenoiser#8]
Hello, can I get the export.py file of onnx?

**Comment by yuyun2000** [comment]
Which model do you want, 48k or 16k?

**Comment by ccf854** [comment]
> Which model do you want, 48k or 16k?

48k，very thanks!

**Comment by yuyun2000** [comment]
try it:https://github.com/grazder/DeepFilterNet/blob/torchDF-changes/torchDF/model_onnx_export.py

**Comment by ccf854** [comment]
> try it:https://github.com/grazder/DeepFilterNet/blob/torchDF-changes/torchDF/model_onnx_export.py
Have you ever encountered problems in transferring onnx files to rknn files when atten_lim_db has topology sorting problems?

**Comment by yuyun2000** [comment]
Yes, it cannot be directly converted to rknn, special processing is required

**Comment by ccf854** [comment]
> Yes, it cannot be directly converted to rknn, special processing is required

Oh，do you have any good solutions?

**Comment by yuyun2000** [comment]
No, I don't have any relevant needs, so I didn't continue.

**Comment by ccf854** [comment]
> No, I don't have any relevant needs, so I didn't continue.

Hello, I bother you again, please ask if you have 16k C++ copy code, my C++ stft features are always different from python, although they are the same operation

**Comment by yuyun2000** [comment]
I don't have it either. I suggest you ask in the gtcrn repository. Someone there has written the reasoning for C, or tell them your question.
